plugins/optimization-detective/optimization.php

```diff
@@ -170,6 +170,10 @@ function od_is_response_html_content_type(): bool {
  * @since 0.1.0
  * @access private
  *
+ * @global WP_Query $wp_the_query            WP_Query object.
+ * @global string   $_wp_current_template_id Current template ID.
+ * @global string   $template                Template file path.
```

Comment:
In the function code below, perhaps better to also add `global $wp_the_query` et al instead of using the `$GLOBALS` array.